apps/src/gamelab/AnimationTab/PiskelEditor.jsx

```diff
@@ -65,6 +70,39 @@ const PiskelEditor = React.createClass({
     if (newProps.selectedAnimation !== this.props.selectedAnimation) {
       this.loadSelectedAnimation_(newProps);
     }
+    if (newProps.pendingAnimationAddition
```

Comment:
nit, i believe our style guide prefers the && to be on the preceding line, rather than starting a new one. also, it should be indented 4 spaces (to differentiate it from the code inside the block on line 75)